tools/setup/emoji/build_emoji

```diff
@@ -262,6 +263,20 @@ def dump_emojis(cache_path):
             os.path.join(cache_emoji, fn)
         )
 
+    # Copy individual emoji images to be used in email notifications.
+    src_emoji_farm = os.path.join(NODE_MODULES_PATH, 'emoji-datasource-google', 'img', 'google', '64', '*')
+    target_emoji_farm = os.path.join(cache_path, 'images-google-64')
+    run(['mkdir', target_emoji_farm])
+    run(['cp', '-RPp', src_emoji_farm, target_emoji_farm], shell=True)
+
+    # The following two images are not present in the new google emoji image farm but were present
+    # in the old emoji farm. So we need to separately copy these images to target dump.
+    old_emoji_farm = os.path.join(cache_path, 'images', 'emoji', 'unicode')
+    missing_image_files = ['1f48f.png', '1f491.png']
```

Comment:
@timabbott If we merge this separately, we will need to keep this piece of code even after the migration to iamcal's dataset is complete which I kinda want to avoid. So would you be merging this separately or should I include this PR in the emoji web PR?

Reply:
I'd like to be able to merge this separately.  But I'm not sure why we wouldn't need this in any case; is the idea that with the new emoji system, it won't be possible to generate these emoji, so we don't need them?

Reply:
Yeah it will not be possible to generate these two emojis as we use only those emojis in the new system whose images are present in all the four emoji sets.